pom.xml

```diff
@@ -86,7 +94,7 @@
     -->
 
     <!--Dependency versions-->
-    <arpnetworking.commons.version>1.3.0</arpnetworking.commons.version>
+    <arpnetworking.commons.version>1.7.1</arpnetworking.commons.version>
```

Comment:
I didn't change this.  Its possible the change ended up as an artifact of moving my PR to another repo.